src/github.rs

```diff
@@ -213,61 +214,19 @@ pub struct Contents {
 	pub download_url: String,
 }
 
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
-#[serde(rename_all = "snake_case")]
-pub enum Event {
-	AddedToProject,
-	Assigned,
-	AutomaticBaseChangeSucceeded,
-	Closed,
-	CommentDeleted,
-	ConvertedNoteToIssue,
-	ConvertToDraft,
-	Demilestoned,
-	HeadRefDeleted,
-	HeadRefForcePushed,
-	HeadRefRestored,
-	Labeled,
-	Locked,
-	Mentioned,
-	MarkedAsDuplicate,
-	Merged,
-	Milestoned,
-	MovedColumnsInProject,
-	ReadyForReview,
-	Referenced,
-	RemovedFromProject,
-	Renamed,
-	Reopened,
-	ReviewDismissed,
-	ReviewRequested,
-	ReviewRequestRemoved,
-	Subscribed,
-	Transferred,
-	Unassigned,
-	Unlabeled,
-	Unlocked,
-	UnmarkedAsDuplicate,
-	Unsubscribed,
-	UserBlocked,
+#[derive(
+	Debug, Clone, PartialEq, Serialize, Deserialize, strum_macros::ToString,
```

Comment:
only `ToString` is actually needed? serde is unused, but you could generate `ToString` with serde as well

Reply:
> only ToString is actually needed?

So far, yes

>  you could generate ToString with serde as well

Couldn't find a way to do that. Please post a coded suggestion.

Reply:
you could use `serde_json::to_string`, but I realized, you could also just implement a enum with unknown field:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=dbda67d6e5004041d932a0526764362a

Reply:
Suggestion taken. Thanks for the review!